sdk/transport_policies/curl/test/main.c

```diff
@@ -0,0 +1,159 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@antkmsft were these supposed to be deleted? i think my merge conflict resolution brought it back

Reply:
removed

Reply:
You're right, sdk/transport_policies/curl is now platform/http_client/curl. But we deleted the test .c for az_curl lib entirely.